### PR TITLE
Add DOI badge

### PR DESCRIPTION
**Pull request by veroandreo**
I have added the DOI badge for 8.2.1 in the Licence page and also included the doi in the bibtex string. Would be nice to somehow automate it though

image

### CODE REVIEW

**Comment by veroandreo** [comment]
Now I also added DOI to the main page and changed the link to point to zenodo instead of brand page

image

**Comment by neteler** [comment]
Looks goot me to.

> Would be nice to somehow automate it though

Not sure what you refer to?

File: content/about/license.md [hunk 1]
```diff
@@ -56,7 +58,8 @@ scientific work written in LaTeX.</p>
     organization = {Open Source Geospatial Foundation},
     address = {USA},
     year = {2022},
-    url = {https://grass.osgeo.org}
+    url = {https://grass.osgeo.org},
+    doi = {10.5281/zenodo.7764250}
```
[the remark this answers is not in the document]
Comment (veroandreo) [reply content/about/license.md:62]: Thanks @nobeeakon, I'll have a look at this and try... But this sounds what I wanted indeed, so when a new version is released and gets a new doi, we only update the variable...

[the remark this answers is not in the document]
Comment (petrasovaa) [reply content/about/license.md:62]: We can use the stable DOI https://doi.org/10.5281/zenodo.5176030 that points to the latest release (the last release done, not release with highest version number). So there shouldn't be any reason to change that, unless we want for the DOI to always refer to the release with highest revision number.

[the remark this answers is not in the document]
Comment (veroandreo) [reply content/about/license.md:62]: Yes! I was searching for this one originally, now I see I didn't see the fine print... eheh

File: content/about/license.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ See also the [Frequently Asked Questions about the GNU GPL](https://www.gnu.org/
 
 ### Citing GRASS GIS Software
 
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7764250.svg)](https://doi.org/10.5281/zenodo.7764250)
```
Comment (veroandreo) [inline content/about/license.md:36]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5176030.svg)](https://doi.org/10.5281/zenodo.5176030)
```

File: content/about/license.md (hunk 1 above)
Comment (veroandreo) [inline content/about/license.md:62]:
```suggestion
    doi = {10.5281/zenodo.5176030}
```

File: themes/grass/layouts/index.html [hunk 3]
```diff
@@ -84,7 +84,10 @@ <h3 class="entry-title" itemprop="headline"><a class="nws" href="{{.RelPermalink
             {{ partial "osgeo.html" . }}
         </div>
         <div class="col-lg-4 col-sm-6 mb-4">
-            <a class="px-4 py-5 bg-gray text-center shadow d-block fdiv" href="{{.Site.BaseURL}}/about/brand"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grasslogo.svg" width="39%"></a>
+            <a class="px-4 py-5 bg-gray text-center shadow d-block fdiv" href="https://doi.org/10.5281/zenodo.7764250">
+                <img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grasslogo.svg" width="31%"><br>
+                <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.7764250.svg" alt="DOI">
+            </a>
```
Comment (veroandreo) [inline themes/grass/layouts/index.html:90]:
```suggestion
            <a class="px-4 py-5 bg-gray text-center shadow d-block fdiv" href="https://doi.org/10.5281/zenodo.5176030">
                <img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grasslogo.svg" width="31%"><br>
                <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.5176030.svg" alt="DOI">
            </a>
```

**Review by veroandreo — COMMENTED** [review]
self review :P

**Pull request merged**